### PR TITLE
feat: [Collection Stats] update listings copy and change value to percentage

### PR DESCRIPTION
**Pull request by gbugyis**
- updates Collection Stats 'Listings' copy text ('Listings' -> 'Listed') and changes the value from a raw number to percentage of total supply.

### CODE REVIEW

File: src/nft/components/collection/CollectionStats.tsx [hunk 1]
```diff
@@ -240,7 +240,9 @@ const StatsItem = ({ children, label, isMobile }: { children: ReactNode; label:
 const StatsRow = ({ stats, isMobile, ...props }: { stats: GenieCollection; isMobile?: boolean } & BoxProps) => {
   const numOwnersStr = stats.stats ? putCommas(stats.stats.num_owners) : 0
   const totalSupplyStr = stats.stats ? putCommas(stats.stats.total_supply) : 0
-  const totalListingsStr = stats.stats ? putCommas(stats.stats.total_listings) : 0
+  const listedPercentageStr = stats.stats
+    ? ((stats.stats.total_listings / stats.stats.total_supply) * 100).toFixed(1)
```
Comment (cbachmeier) [inline src/nft/components/collection/CollectionStats.tsx:244]: need a check to make sure `stats.stats.total_listings` is greater than 0

File: src/nft/components/collection/CollectionStats.tsx [hunk 2]
```diff
@@ -267,8 +269,8 @@ const StatsRow = ({ stats, isMobile, ...props }: { stats: GenieCollection; isMob
         </StatsItem>
       ) : null}
       {stats.stats?.total_listings ? (
```
Comment (cbachmeier) [inline src/nft/components/collection/CollectionStats.tsx:271]: Can you check with product to see if we want to show when there's `0%` listed?

File: src/nft/components/collection/CollectionStats.tsx [hunk 3]
```diff
@@ -267,8 +269,8 @@ const StatsRow = ({ stats, isMobile, ...props }: { stats: GenieCollection; isMob
...
-        <StatsItem label="Listings" isMobile={isMobile ?? false}>
-          {totalListingsStr}
+        <StatsItem label="Listed" isMobile={isMobile ?? false}>
+          {listedPercentageStr} %
```
Comment (cbachmeier) [inline src/nft/components/collection/CollectionStats.tsx:273]: I think we should get rid of the space before the `%` which would be in parity with our percentage behaviour elsewhere on the app

**Review by cbachmeier — CHANGES_REQUESTED** [review]

File: src/nft/components/collection/CollectionStats.tsx (hunk 2 above)
Comment (gbugyis) [reply src/nft/components/collection/CollectionStats.tsx]: Hiding it for now until I get confirmation

**Review by cbachmeier — APPROVED** [review]
LGTM

**Pull request merged**